Probesys/bileto
# Imp/permissions confidential messages

## Related issue(s)



https://github.com/Probesys/bileto/issues/210

## Changes



- Add two permissions: one to show confidential messages, the other to see confidential messages
- Filter the confidential messages if authorization is not granted via a new `Ticket#getMessagesWithoutConfidential` method
- Hide confidential checkbox if authorization is not granted, and check the permission on the server as well

## How to test manually



- create two roles: one with the mentioned permissions, the other without them
- post a confidential message with the first role and check you can see the answer
- try to post a confidential message with the second role, check the checkbox doesn't appear

## Checklist
